src/Query/Processor.php

```diff
@@ -73,30 +73,25 @@ protected function processColumn(mixed $value): mixed
      * Process the results of a columns query.
      *
      * {@inheritDoc}
-     * @param array<array-key, array<array-key, mixed>> $results
-     * @return array<array-key, array{
-     *     name: string,
-     *     type_name: string,
-     *     type: string,
-     *     collation: null,
-     *     nullable: bool,
-     *     default: scalar,
-     *     auto_increment: false,
-     *     comment: null
-     * }>
+     * @param list<array<string, mixed>> $results
+     * @return list<array{name: string, type: string, type_name: string, nullable: bool, collation: string|null, default: mixed, auto_increment: bool, comment: string|null, generation: array{type: string, expression: string|null}|null}>
```

Comment:
Why is the return type ambiguous for some attributes?
For example, `collation` is `string|null`, but it returns always `null`.

Reply:
It's because I copied it from laravel 12's phpdoc (which contains types).
This is now fixed.